src/MOI_wrapper.jl

```diff
@@ -431,5 +433,107 @@ end
+infeasible.
+
+See also `CPLEX.ConflictStatus`, `CPLEX.VariableConflictStatus`, and
+`CPLEX.ConstraintConflictStatus`.
+
+Note that if `model` is modified after a call to `compute_conflict`, the
+conflict is not purged, and any calls to the above attributes will return values
+for the original conflict without a warning.
+"""
+function compute_conflict(model::Optimizer)
+    model.conflict = c_api_getconflict(model.inner)
+    return
+end
+
+function _ensure_conflict_computed(model::Optimizer)
+    if model.conflict === nothing
+        error("Cannot access conflict status. Call `CPLEX.compute_conflict(model)` first. " * 
+              "In case the model is modified, the computed conflict will not be purged.")
+    end
+end
 
+"""
+    ConflictStatus()
+
+Return an `MOI.TerminationStatusCode` indicating the status of the last computed conflict.
+
+If a minimal conflict is found, it will return `MOI.OPTIMAL`. If the problem is feasible, it will
+return `MOI.INFEASIBLE`. If `compute_conflict` has not been called yet, it will return 
+`MOI.OPTIMIZE_NOT_CALLED`. 
+"""
+struct ConflictStatus <: MOI.AbstractModelAttribute  end
+MOI.is_set_by_optimize(::ConflictStatus) = true
+
+function MOI.get(model::Optimizer, ::ConflictStatus) 
+    if model.conflict === nothing
+        return MOI.OPTIMIZE_NOT_CALLED
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_MINIMAL
+        return MOI.OPTIMAL
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_FEASIBLE
+        return MOI.INFEASIBLE
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_CONTRADICTION
+        return MOI.OTHER_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_DETTIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_IT_LIM
+        return MOI.ITERATION_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_MEM_LIM
+        return MOI.MEMORY_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_NODE_LIM
+        return MOI.NODE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_OBJ_LIM
+        return MOI.OBJECTIVE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_TIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_USER
+        return MOI.OTHER_LIMIT
+    else
+        return MOI.OTHER_LIMIT
+    end
+end
+
+function MOI.supports(::Optimizer, ::ConflictStatus)
+    return true
+end
+
+"""
+    ConstraintConflictStatus()
+
+A Boolean constraint attribute indicating whether the constraint participates in the last computed conflict.
+"""
+struct ConstraintConflictStatus <: MOI.AbstractConstraintAttribute end
+MOI.is_set_by_optimize(::ConstraintConflictStatus) = true
+
+function MOI.get(model::Optimizer, ::ConstraintConflictStatus, index::MOI.ConstraintIndex) 
+    _ensure_conflict_computed(model)
+    return LQOI.get_column(model, index) in model.conflict.rowind
```

Comment:
I think you need something like
```julia
function MOI.get(model::Optimizer, ::ConstraintConflictStatus, 
                 index::MOI.ConstraintIndex{<:MOI.ScalarAffineFunction, <:LQOI.LinSets}) 
    _ensure_conflict_computed(model)
    return model[index] in model.conflict.rowind
end
```

Reply:
It does not work properly: for the test case I've written, `model[index]` returns `MathOptInterface.VariableIndex(1)`, i.e. the lhs of the constraint, I guess. I've tried to enumerate the cases and write specific methods that use `cmap`, but it's still not working… I'm still investigating other ways to have it work correctly.

Reply:
That's because your test case uses `SingleVariable` constraints. See my comment https://github.com/JuliaOpt/CPLEX.jl/pull/233#discussion_r277172977. 

The above will only work for `MOI.ScalarAffineFunction` constraints as I put in the function header.

For `MOI.SingleVariable` constraints, you can go
```julia
function MOI.get(model::Optimizer, ::ConstraintConflictStatus, 
                 index::MOI.ConstraintIndex{<:MOI.SingleVariable, <:LQOI.LinSets}) 
    return MOI.get(model, MOI.VariableConflictStatus(), model[index])
end
```